README.md

```diff
@@ -1,60 +1,71 @@
-[![Saleor](http://getsaleor.com/mr-saleor-readme.png)](http://getsaleor.com)
-
-
-Saleor
-======
-
-Avast ye landlubbers! Saleor is an open-source e-commerce storefront for Python and Django.
-
-[Saleor homepage](http://getsaleor.com/)
+# Saleor
 
 [![Build Status](https://travis-ci.org/mirumee/saleor.svg?branch=master)](https://travis-ci.org/mirumee/saleor)
-[![Requirements Status](https://requires.io/github/mirumee/saleor/requirements.svg?branch=master)](https://requires.io/github/mirumee/saleor/requirements/?branch=master)
 [![codecov.io](http://codecov.io/github/mirumee/saleor/coverage.svg?branch=master)](http://codecov.io/github/mirumee/saleor?branch=master)
+[![Join the chat at https://gitter.im/mirumee/saleor](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/mirumee/saleor?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Twitter URL](https://img.shields.io/twitter/url/https/twitter.com/fold_left.svg?style=social&label=Follow%20%40getsaleor)](https://twitter.com/getsaleor)
 
-* * *
 
-Your feedback
--------------
+Saleor is high-performance e-commerce solution built with Python and Django.
 
-Do you use Saleor as an e-commerce platform?
-Fill out this short survey and help us grow. It will take just a minute, but mean a lot!
+We believe that better architecture means productive developers, trying to keep it simple, lightweight, and modular.
 
-[Survey](https://mirumee.typeform.com/to/sOIJbJ)
+Service-based architecture, top-notch technologies. Django, PostgreSQL, ElasticSearch, GraphQL and Docker.
 
-* * *
+[Visit our homepage to find out more](http://getsaleor.com/)
 
-Usage
------
+# Getting started
 
-See the [Saleor docs](https://saleor.readthedocs.io) for installation and deployment instructions.
+## 💾 Installation and requirements
 
+Saleor requires Python 3.4+, Node.js, PostgreSQL and OS-specific dependency tools.
```

Comment:
I think you should say Node.js 8.0+, especially as xenial and others are still on 4.0+ which is often error prone for many users.